src/vm/comutilnative.cpp

```diff
@@ -826,13 +826,6 @@ void QCALLTYPE Buffer::MemMove(void *dst, void *src, size_t length)
 {
     QCALL_CONTRACT;
 
-#if !defined(FEATURE_CORESYSTEM)
```

Comment:
This is just removing dead code because `FEATURE_CORESYSTEM` is always defined for CoreCLR.